XCUITests/SaveLoginsTests.swift

```diff
@@ -68,55 +134,50 @@ class SaveLoginTest: BaseTestCase {
         XCTAssertTrue(app.cells.images["loginUnselected"].exists)
     }
 
-
     func testDeleteLogin() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
         openLoginsSettings()
-        app.tables.cells[savedLoginEntry].tap()
+        app.staticTexts[domain].tap()
         app.cells.staticTexts["Delete"].tap()
         waitForExistence(app.alerts["Are you sure?"])
         app.alerts.buttons["Delete"].tap()
         waitForExistence(app.tables["Login List"])
-        XCTAssertFalse(app.tables.cells[savedLoginEntry].exists)
-        XCTAssertEqual(app.tables["Login List"].cells.count, 0)
+        XCTAssertFalse(app.staticTexts[domain].exists)
+        XCTAssertFalse(app.staticTexts[domainLogin].exists)
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsEmptyLoginsList)
         XCTAssertTrue(app.tables["No logins found"].exists)
     }
 
-
     func testEditOneLoginEntry() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
         openLoginsSettings()
-        XCTAssertTrue(app.tables.cells[savedLoginEntry].exists)
-
-        app.tables.cells[savedLoginEntry].tap()
+        XCTAssertTrue(app.staticTexts[domain].exists)
+        XCTAssertTrue(app.staticTexts[domainLogin].exists)
+        app.staticTexts[domain].tap()
         waitForExistence(app.tables["Login Detail List"])
-        XCTAssertTrue(app.tables.cells["website, \(domain)"].exists)
-        XCTAssertTrue(app.tables.cells["username, test@example.com"].exists)
-        XCTAssertTrue(app.tables.cells["password"].exists)
+        XCTAssertTrue(app.tables.cells[loginsListURLLabel].exists)
```

Comment:
As commented above there seems to be IDs for the elements in the Logins Detail View although they are not shown, once that is clarified, we could check that the elements are there with the IDs, and the content of each one as you are doing.